docs/core/actions.rst

```diff
@@ -228,9 +228,10 @@ There are eight default actions:
 |                                   | events. This action is triggered automatically |
 |                                   | after an inactivity period defined by the      |
 |                                   | ``session_expiration_time`` parameter in the   |
-|                                   | domain's session config. Can be triggered      |
-|                                   | manually during a conversation by entering     |
-|                                   | ``/session_start``.                            |
+|                                   | domain's :ref:`session_config`. Can be         |
```

Comment:
L214 -- can you change to 9? Or just remove/rephrase?

Reply:
👍